pimd/pim_rp.c

```diff
@@ -1280,11 +1302,19 @@ void pim_rp_show_information(struct pim_instance *pim, struct vty *vty, bool uj)
 
 				json_object_array_add(json_rp_rows, json_row);
 			} else {
+#if PIM_IPV == 4
+				vty_out(vty, "%-15s  ",
+					inet_ntop(
```

Comment:
Use %pFX and pass &rp_info->rp.rpf_addr, for ex:

vty_out(vty, "%-15pFX ",  rp_info->rp.rpf_addr); Then you don't need #if and #else. It will be common code.